### PR TITLE
solve strange issues occurred in building golden image and invoking SequenceR

### PR DESCRIPTION
**Pull request by jaxxgu**
build on __pytorch__ image instead of __ubuntu__ image
add the link of existing third-party SequenceR docker image
add necessary hints about installing GitLFS
add more output for executing `sequencer-predict.sh`
fix the steps of installing __`OpenNMT-py`__ in `setup_env.sh`

### CODE REVIEW

File: src/setup_env.sh [hunk 1]
```diff
@@ -48,9 +48,12 @@ fi
 
 if [ ! -d $CURRENT_DIR/lib/OpenNMT-py ]; then
   echo "Cloning OpenNMT-py"
-  git clone https://github.com/chenzimin/OpenNMT-py.git $CURRENT_DIR/lib/OpenNMT-py
+  # git clone https://github.com/chenzimin/OpenNMT-py.git $CURRENT_DIR/lib/OpenNMT-py
+  git clone https://github.com/OpenNMT/OpenNMT-py.git $CURRENT_DIR/lib/OpenNMT-py
   echo "Installing requirements for OpenNMT-py"
-  pip3 install -r $CURRENT_DIR/lib/OpenNMT-py/requirements.txt
+  cd $CURRENT_DIR/lib/OpenNMT-py
+  pip install -r requirements.txt
```
Comment (chenzimin) [inline src/setup_env.sh]: Can you use pip3 here

File: src/setup_env.sh [hunk 2]
```diff
@@ -48,9 +48,12 @@ fi
...
+  python setup.py install --force
```
Comment (chenzimin) [inline src/setup_env.sh]: Can you use python3 here

Comment (jaxxgu) [reply src/setup_env.sh]: I tried to use Python3 here but then it fails in installing libs.

File: src/setup_env.sh (hunk 1 above)
Comment (jaxxgu) [reply src/setup_env.sh]: I tried to use `pip3` here but then it fails in installing libs.

Comment (jaxxgu) [reply src/setup_env.sh]: It usually fails at the step of installing __`torch`__ lib so I learn from [OpenNMT-py's Dockerfile](https://github.com/OpenNMT/OpenNMT-py/blob/master/Dockerfile#L2)

File: src/setup_env.sh (hunk 2 above)
Comment (jaxxgu) [reply src/setup_env.sh]: As I have to install libs via `pip3`, I use `python3` here

File: src/setup_env.sh [hunk 3]
```diff
@@ -48,9 +48,12 @@ fi
 
 if [ ! -d $CURRENT_DIR/lib/OpenNMT-py ]; then
   echo "Cloning OpenNMT-py"
-  git clone https://github.com/chenzimin/OpenNMT-py.git $CURRENT_DIR/lib/OpenNMT-py
+  # git clone https://github.com/chenzimin/OpenNMT-py.git $CURRENT_DIR/lib/OpenNMT-py
+  git clone https://github.com/OpenNMT/OpenNMT-py.git $CURRENT_DIR/lib/OpenNMT-py
```
Comment (chenzimin) [inline src/setup_env.sh]: Why is this change necessary? Because we have no guarantee that OpenNMT won't change it CLI and the options. Therefore it is better to clone one fixed version here

File: src/setup_env.sh (hunk 1 above)
Comment (chenzimin) [reply src/setup_env.sh]: I would like to keep it consistent with the rest (always using python3 and pip3), and this file can be used without using docker. Can you build a image without clone and install OpenNMT, and try install it manually with python3 and pip3 to see the error message?

File: src/setup_env.sh (hunk 3 above)
Comment (jaxxgu) [reply src/setup_env.sh]: I agree. Here it gets back to the original state.

File: src/setup_env.sh (hunk 1 above)
Comment (jaxxgu) [reply src/setup_env.sh]: Yes, I tried. Then `pip3 install -r ...` would always be killed automatically. I also tried to wget the .whl file of __`torch`__ and then install it individually, also failed.

Comment (chenzimin) [reply src/setup_env.sh]: What error message is displayed when it is killed?

Comment (jaxxgu) [reply src/setup_env.sh]:
```
Collecting torch>=1.1 (from -r requirements.txt (line 3))
  Downloading https://files.pythonhosted.org/packages/30/57/d5cceb0799c06733eefce80c395459f28970ebb9e896846ce96ab579a3f1/torch-1.2.0-cp36-cp36m-manylinux1_x86_64.whl (748.8MB)
./src/setup_env.sh: line 58:    38 Killed                  pip3 install -r requirements.txt
```

Comment (chenzimin) [reply src/setup_env.sh]: So I tried it locally (building image without OpenNMT and run pip3 install), it works fine.

When searching for these errors, I found https://github.com/pytorch/pytorch/issues/1022 and https://stackoverflow.com/questions/43245196/pip-install-killed

Comment (jaxxgu) [reply src/setup_env.sh]: Hey @chenzimin 
Yes, you are right. But for my work of integrating SequenceR into Repairnator, it is better to have everything within one docker image. I just need to invoke SequenceR and get correct results. I totally agree that it is bad practice to use __Python 2__ in 2019, but from another perspective, we are just following steps from OpenNMT-py's Dockerfile, and it is more reliable to keep consistent with upstreams.

We can add more content to `README.md` if necessary.

Comment (chenzimin) [reply src/setup_env.sh]: I am not saying using python2 is bad here, but merely that it seems to be an out of memory issue instead.

Comment (chenzimin) [reply src/setup_env.sh]: Can you try https://stackoverflow.com/questions/44533319/how-to-assign-more-memory-to-docker-container/44533437#44533437, or just forked this project for your work? You can still sync to this project by https://help.github.com/en/articles/syncing-a-fork

Comment (jaxxgu) [reply src/setup_env.sh]: If I am using your way to build image, it works. But I can not get correct results by invoking SequenceR image.

Comment (jaxxgu) [reply src/setup_env.sh]: I know it is one OOM issue.

Comment (jaxxgu) [reply src/setup_env.sh]: Or maybe you could release one official SequenceR image to docker hub, then we do not have to make these modifications on this repo.

Comment (jaxxgu) [reply src/setup_env.sh]: > Can you try https://stackoverflow.com/questions/44533319/how-to-assign-more-memory-to-docker-container/44533437#44533437, or just forked this project for your work? You can still sync to this project by https://help.github.com/en/articles/syncing-a-fork

Yes, I increased memory size by this way and it did works. What I mean is that, I still meet other issues so that I can not utilize SequenceR normally only by invoking its image. In order to build one effective image, I have to make such changes to these files.

Comment (chenzimin) [reply src/setup_env.sh]: Let's us discuss in the github issue instead

**Comment by chenzimin** [comment]
Close pull request per request from https://github.com/KTH/sequencer/issues/2
